test/unit/event_machine_proxy_test.rb

```diff
@@ -89,19 +88,19 @@
     end
 
     it "add_timer" do
-      mock(ExceptionHandling).log_error(EXCEPTION, "add_timer")
+      mock(ExceptionHandling).log_error(EXCEPTION, "add_timer", {})
```

Comment:
I did a quick inspection of the exception_handling gem and I believe the empty hash is now passed due to changes for contextual logging. Perhaps you can confirm?